client/src/components/pages/performanceReview.js

```diff
@@ -1,9 +1,18 @@
-import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+import SideBar from "../parts/sidebar";
+import NavBar from "../parts/navbar";
+import PartContainer from "../parts/partContainer"
 
-export default class performanceReview extends Component {
-  render() {
-    return (
-      <h1>performanceReview</h1>
-    )
-  }
-}
\ No newline at end of file
+const PerformanceReview = () => {
+  return (
+    <div>
+      {/* TODO: when I change title = Performance Review, it gets rid of some 
```

Comment:
@GregoryGarber when I put the full text of "Performance Review" the navbar breaks cause its too long, was thinking we can adjust the font size for this specific title in a later commit